### ISSUE TITLE
The documentation for the services markdown is wrong

### ISSUE THREAD

**Issue by Chucky1971** [dummylabs/thewatchman#112]
The documentation for the services markdown is wrong, please adjust.

In documentation

```
_type: markdown
content: >-
  <h2 class="some"> <ha-icon icon='mdi:shield-half-full'></ha-icon> Watchman report</h2>
  <h3> Missing Services: {{ states.sensor.watchman_missing_services.state }} </h3>
  {%- for item in state_attr("sensor.watchman_missing_services", "services") %}
  <hr><table><tr> <td>  <ha-icon icon='mdi:cloud-alert'></ha-icon> {{ item.id }}
  <a title="{{item.occurrences}}">{{item.occurrences.split('/')[-1].split(':')[0]}}</a>
  </td></tr></table>
  {%- endfor %}_
```
  
  "services" should be "entities" to work:

```
_type: markdown
content: >-
  <h2 class="some"> <ha-icon icon='mdi:shield-half-full'></ha-icon> Watchman report</h2>
  <h3> Missing Services: {{ states.sensor.watchman_missing_services.state }} </h3>
  {%- for item in state_attr("sensor.watchman_missing_services", "entities") %}
  <hr><table><tr> <td>  <ha-icon icon='mdi:cloud-alert'></ha-icon> {{ item.id }}
  <a title="{{item.occurrences}}">{{item.occurrences.split('/')[-1].split(':')[0]}}</a>
  </td></tr></table>
  {%- endfor %}_
```

**Comment by ildar170975** [comment]
This issue is the THIRD about the same.
This is what happens when people do not use "search" before posting.

**Comment by dummylabs** [comment]
Should be fixed together with #52 and #93 in 0.6.2
